### PR TITLE
add prometheus remote write project for june 2024

### PR DESCRIPTION
**Pull request by cstyan**
Project for upcoming summer session related to benchmark testing of Prometheus Remote Write. Not sure if @jesusvazquez is already registered on the LFX mentorship platform.

### CODE REVIEW

**Review by nate-double-u — CHANGES_REQUESTED** [review]
Thanks for this @cstyan!

Could you provide an upstream issue as well? This helps a couple ways: when I load this into LFX I need a link to the projcect, and it gives folks a place to ask questions or comment on the work.

Also, @jesusvazquez, could you give a 👍 or approve this PR as your confirmation of wanting to participate? (Nico & Alex, you too--I just don't have your GitHub handles to tag you 🙂)

**Comment by cstyan** [comment]
cc @npazosmendez @alexgreenbank

**Comment by cstyan** [comment]
> Could you provide an upstream issue as well? This helps a couple ways: when I load this into LFX I need a link to the projcect, and it gives folks a place to ask questions or comment on the work.

Done :+1:  it didn't exist before so I just created one under the umbrella of our larger remote write 2.0 work meta issue as that's the original context this project came up in.

Tagged Nico and Alex as well. They'll likely be working on other portions of the remote 2.0 implementation itself, and so will have less time for direct mentoring but just as much context around the use case of what the mentee would be building. Jesús has context around prombench itself.

File: programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md [hunk 1]
```diff
@@ -30,4 +30,16 @@
...
+- Expected Outcome: Build additional (or extends existing) tooling, similar to Prombench’s [load-generator](https://github.com/prometheus/test-infra/tree/master/tools/load-generator) and [avalanche](https://github.com/prometheus-community/avalanche), to support scenarios under which remote write should be performance tested. For example; allowing gradual increases/decreases in # of active series, sudden spikes in active series, various amounts of latency in the server receiving the remote write data, etc. Time permitted, extend Prombench's test suite to include a set of Remote Write tests that can be run via a new command.
+- Recommended Skills: Go, some familiarity with Prometheus or metrics
+- Mentor(s): 
+  - Callum Styan (@cstyan, callumstyan@gmail.com), 
+  - Jesús Vázquez (@jesusvazquez, jesus.vazquez@grafana.com)
+  - Nico Pazos and Alex Greenbank from Grafana also available to help
+
```
Comment (aliok) [inline programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md:44]:
```suggestion
- Upstream Issue: https://github.com/...
```

@cstyan can you add the upstream issue here please?

**Comment by alexgreenbank** [comment]
:+1: Happy to be involved!

**Review by jesusvazquez — APPROVED** [review]
LGTM

**Comment by jesusvazquez** [comment]
Hey @cstyan I'm registered in the LFX platform already 👍 

@nate-double-u just approved 👍

**Comment by npazosmendez** [comment]
Happy to be involved too!

File: programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md (hunk 1 above)
Comment (nate-double-u) [reply programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md:44]: > I just created one under the umbrella of our larger remote write 2.0 work meta issue as that's the original context this project came up in.

If you can drop it in here I can merge it in 🙂

Comment (cstyan) [reply programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md:45]: my bad, made the change locally last night but forgot to pushy, the link is there now

**Review by nate-double-u — APPROVED** [review]
Thanks everyone!

**Pull request merged**